src/status_im/ui/screens/keycard/components/keycard_animation.cljs

```diff
@@ -0,0 +1,336 @@
+        card-loop        (animation/anim-loop
+                          (animation/anim-sequence
+                           [(animation/timing card
+                                              {:toValue  #js {:x -30
+                                                              :y 30}
+                                               :duration 1000
+                                               :easing   card-easing})
+                            (animation/timing card
+                                              {:toValue  {:x 45
+                                                          :y 65}
+                                               :duration 1000
+                                               :delay    2000
+                                               :easing   card-easing})
+                            (animation/timing card
+                                              {:toValue  #js {:x -30
+                                                              :y 105}
+                                               :duration 1000
+                                               :delay    2000
+                                               :easing   card-easing})
+                            (animation/anim-delay 2000)]))
+        phone-entrace    (animation/anim-sequence
+                          [(animation/anim-delay phone-enter-at)
+                           (animation/parallel
+                            [(animation/timing card-scale
+                                               {:toValue  0.528
+                                                :duration 1000
+                                                :easing   card-easing})
+                             (animation/timing phone
+                                               {:toValue  1
+                                                :duration 1000
+                                                :easing   card-easing})
+                             card-loop])])
+        circles          (animation/anim-loop
+                          (animation/parallel
+                           [(animation/anim-sequence
+                             [(circle-animation small 1 0)
+                              (circle-animation small 0 0)])
+                            (animation/anim-sequence
+                             [(circle-animation medium 1 200)
+                              (circle-animation medium 0 0)])
+                            (animation/anim-sequence
+                             [(circle-animation big 1 400)
+                              (circle-animation big 0 0)])]))
+        animation        (animation/parallel
+                          [resets
+                           phone-entrace
+                           circles])]
+    (reset! state :init)
+    ;; TODO: Think how to improve that, this creates desync with animation
+    (js/setTimeout #(compare-and-set! state :init :awaiting)
```

Comment:
Suggestion on how to improve that are welcome 🙏

Reply:
The only option I see here is to start `phone-entrance` as a separate animation and to reset `state` right before starting this animation. But it wouldn't look much better than it is now and will require usage of `setTimeout` anyway. Though probably changing of `state` and animation would be synchronised better.